comma_hack_4

nuwandavek opened the pull request:
- [x] Make a good README to get started
- [x] Move all body controls from web (it should only do WASD)
- [x] Add remote vipc  + model inference
- [x] Add yolo
- [ ] Consume yolo outputs in bodycontrolsd
- [ ] remote ip on web ui?

mitchellgoffpc committed:
Added bodycontrolsd.py

mitchellgoffpc committed:
Bump cereal

mitchellgoffpc committed:
Bug fixes

Commit:
remove non waSD FROM WEB

Commit:
add ratekeeper to bodycontrolsd

Commit:
fix pm error

Commit:
disable locationd, paramsd related alerts for the body

Commit:
log control inputs

Commit:
add logging to bodycontrol

Commit:
add readme

nuwandavek committed:
add yolo, make changed to README

nuwandavek committed:
add todo for remote ip

nuwandavek committed:
move yolo.py

YassineYousfi committed:
resolve yolo todos

YassineYousfi committed:
added some credits

fredyshox committed:
Add cereal bridge section

fredyshox commented on tools/bodyteleop/README.md:
```diff
@@ -0,0 +1,49 @@
+
+### 2. Setup cereal bridge
+
+SSH into the body from the remote computer you intent to run models on. Then start cereal bridge using command below. Replace services... with the list of published sockets you intent to share with the body. For the yolo example that would be: bodyReserved1
```
these are instructions and snippets to get the laptop -> body bridge working. I've created short scripts which does it for you in similar fashion to compressed_vipc.py (you just supply the body ip), but not sure if it's thats necessary

fredyshox committed:
Remove remote_ip from bodycontrolsd

fredyshox committed:
Merge remote-tracking branch 'origin/master' into comma_hack_4

nuwandavek commented on selfdrive/manager/process_config.py:
```diff
@@ -87,6 +87,7 @@ def only_offroad(started, params, CP: car.CarParams) -> bool:
   # debug procs
   NativeProcess("bridge", "cereal/messaging", ["./bridge"], notcar),
```
@fredyshox Can we remove this then?

fredyshox replied:
No, this is bridge for body -> pc communication. The one you need to setup manually is for pc -> body side.

fredyshox committed:
bump cereal

mitchellgoffpc committed:
Bump cereal, text -> data

mitchellgoffpc committed:
Merge branch 'master' into comma_hack_4

Commit:
Added fakemodeld

nuwandavek committed:
make sounds work again

Commit:
Fixed local yolo runner

Commit:
Moved yolo.py to selfdrive/modeld

nuwandavek committed:
change default body volume to 80%

fredyshox committed:
Additional info in README

fredyshox committed:
words

fredyshox committed:
Add note about ports

Commit:
yolo swag

Commit:
hack to patch failing assert during yolo thneed compilation

Commit:
A bit more cleanup

fredyshox committed:
ugly zmq hack

fredyshox committed:
Fix readme

adeebshihadeh committed:
setup ssh

sshane committed:
ssh keys: get multiple users

add to description

update translations

vanish

bye bye translations

clean up

clean up

fix

trim

fredyshox committed:
aarch64: onnxruntime (#30095)

base onnxruntime for aarch64

sshane committed:
disable updates

nuwandavek committed:
remove default keys

mitchellgoffpc committed:
gpt4 is goated with the sauce

fredyshox committed:
Move remote model instructions to hakacthon md

fredyshox committed:
devcontainer: expose zmq ports for bridging (#30072)

* Expose ZMQ ports for bridging

* Limit ports to 8070-8079

fredyshox committed:
Comment for services

mitchellgoffpc committed:
chrome only

fredyshox committed:
devcontainer: use VirtualGL on mac hosts (#30090)

* Install virtualgl in a container

* Initialize virtualgl in bashrc

* Create virtual screen when starting

* Start vglclient on host

* Run Xvfb in separate tmux session to keep it running

* Add note about virtualGL in readme

* Add wget

* source vglrun

* Start xvfb only when forwarding x11

* Remove section about vglrun from readme

* HOST_DISPLAY implementation

* Add MOTD message instead of readme section

mitchellgoffpc committed:
added a note about fakemodeld

mitchellgoffpc committed:
Added a note about ssh

YassineYousfi committed:
nv12 it is
